setup.py

```diff
@@ -7,9 +7,12 @@
 )
 
 tests_require = (
-    'nose'
+    'nose',
 )
 
+setup_requires = (
+    'flake8',
```

Comment:
Interesting. I'd still argue that it should be in `tests_require`, but I can back down given that!